packages/multichain-account-service/package.json

```diff
@@ -86,6 +87,7 @@
   "peerDependencies": {
     "@metamask/account-api": "^0.12.0",
     "@metamask/accounts-controller": "^34.0.0",
+    "@metamask/error-reporting-service": "^3.0.0",
```

Comment:
Since it's a new peer dep, this is a breaking change for the consumers

Reply:
Updated the changelog and title!

Reply:
Nice! Thanks! I did remove the `**BREAKING**` part, since breaking change are denoted with a `!` with conventional commits